src/scanner.cc

```diff
@@ -148,7 +148,7 @@ enum Sym: uint16_t {
   fail,
 };
 
-static vector<string> names = {
+static string names[] = {
```

Comment:
I was also going to suggest this, but then saw you fixed this in a later commit. 
Consider `std::array`?